polymesh_schema.json

```diff
@@ -1357,7 +1357,7 @@
                         "isOptional": true
                     }
                 ],
-                "type": "HistoricalVoting"
+                "type": "HistoricalVotingByAddress"
```

Comment:
I believe this type and the other one isn't defined in the schema, so will need to be added.

Reply:
They already are - https://github.com/PolymathNetwork/Polymesh/blob/2edacd8bb50dd43ce30d91fc4922d56031e5a37f/polymesh_schema.json#L818

Reply:
Odd... I got a different impression from a #substrate_dev convo yesterday 🤦